Src/VimWpf/VimHost.cs

```diff
@@ -255,15 +255,18 @@ public virtual RunCommandResults RunCommand(string command, string arguments, st
             // Use a (generous) timeout since we have no way to interrupt it.
             var timeout = 30 * 1000;
 
+            // Avoid redirection for the 'start' command.
+            var doRedirect = !arguments.StartsWith("/c start ");

```

Comment:
Shouldn't this be a case insensitive comparison?

Reply:
Good point.

Reply:
I made that change.